src/routes/api/departments/[departmentUrl]/challenges/index.ts

```diff
@@ -20,11 +25,14 @@ export async function getChallenges(client: PrismaClient, userId: string, depart
 			lte: new Date()
 		}
 	};
-	if (departmentUrl) {
-		(where as any).department = { url: departmentUrl };
+	if (departmentId) {
+		(where as any).departmentId = departmentId;
```

Comment:
can we not use departmentUrl here?

Reply:
good point, I forgot to switch back to urls here.